contact.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html lang="en-GB">
+
+	<head>
+		<title>Contact Page</title>
+		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
+	</head>
+		
+	<body>
+		
+		<nav class="nav_bar">
+			<p>Karen Schulz</p>
+			
+			<div>
+				<a href="index.html">Home</a>
+				<a href="portfolio.html">Portfolio</a>
+			</div>
+			
+		</nav>
+		
+		<article>
+			<div class="contact">
+				<img class="picture-of-me" src="images/half_body.png" alt="my_profile_picture upper half, with bowler hat inside an antique shop">
+	
+			<div class="hello_stranger">Don't be a stranger and drop me a line!</div>
+			
+			<div class="email_me"><a href=mailto:sirk.schulz@gmail.com>Email Me</a></div>
+	
+			<p>Otherwise you can submit your contact details and I will be in touch:</p>
+		
+		
+		<form>
+			
+			<div class="where">
+				<label>Where did you hear about me?</label>
+		    </div>
+			
+			<p>
+				<label for="friend"><h6>Friend</h6></label>
+				<input type="radio" name="my-radio" id="friend" value="friend" />
+			</p>
+			
+			<p>
+				<label for="githut"><h6>Github</h6></label>
+				<input type="radio" name="my-radio" id="github" value="ngithub" />
+			</p>
+			
+			<div class="checkbox">
+				<label for="my-checkbox">Would you like me to send you my portfolio?</label>
+				<input type="checkbox" name="my-checkbox" value="checkbox" />
+			
+			<div class="enter_email">
+				<label for="email"><h6>Please enter your email:</h6></label>
+				<input type="email"/>
+				<input type="submit" value="submit"/>
+			</div>
+			
+		</form>
+		
+		<div class="links">
+			<a href="index.html">Home</a>
+			<a href="portfolio.html">Portfolio</a>
+		</div>
+		
+		</article> 
+		 s
+	</body>
+	
+	<footer>&copy Copyright. All rights reserved. Karen Schulz, January 2021</footer>
```

Comment:
you need a : after the copy

Reply:
Yes. thanks for reminding me. I already changed it

Reply:
Sorry... got confused... I did the : after the copy and in my browser and it looks wrong, that is why I decided to take it out. Will check other options then, because this is a strange one... Can you see the Copy symbol at all?